azure-cosmos-db/1.0/modules/ROOT/pages/azure-cosmos-db-connector-reference.adoc

```diff
@@ -0,0 +1,1939 @@
+| Response Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit for the *Response Timeout* field. |  |
+| Streaming Type a| Enumeration, one of:
+
+** AUTO
+** ALWAYS
+** NEVER a|  How the connector streams request content to the remote service:
+
+* `AUTO` (default)
+
+Connector automatically uses the best streaming strategy based on the request content.
+
+* `ALWAYS`
+
+Connector always streams the request content.
+
+* `NEVER`
+
+Connector does not stream the request content. |  |
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  |
+| Target Value a| String |  Expression that evaluates the operation's output. The outcome of the expression is stored in the *Target Variable* field. |  #[payload] |
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  |
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Any
+| Attributes Type a| <<HttpResponseAttributes>>
+|===
+
+=== For Configurations
+
+* <<KeyTokenConfig>>
+
+=== Throws
+
+* AZURE-COSMOS-DB-CONNECTOR-MULE-4:BAD_REQUEST
+* AZURE-COSMOS-DB-CONNECTOR-MULE-4:CLIENT_ERROR
+* AZURE-COSMOS-DB-CONNECTOR-MULE-4:CONNECTIVITY
+* AZURE-COSMOS-DB-CONNECTOR-MULE-4:INTERNAL_SERVER_ERROR
+* AZURE-COSMOS-DB-CONNECTOR-MULE-4:NOT_FOUND
+* AZURE-COSMOS-DB-CONNECTOR-MULE-4:RETRY_EXHAUSTED
+* AZURE-COSMOS-DB-CONNECTOR-MULE-4:SERVER_ERROR
+* AZURE-COSMOS-DB-CONNECTOR-MULE-4:SERVICE_UNAVAILABLE
+* AZURE-COSMOS-DB-CONNECTOR-MULE-4:TIMEOUT
+* AZURE-COSMOS-DB-CONNECTOR-MULE-4:TOO_MANY_REQUESTS
+* AZURE-COSMOS-DB-CONNECTOR-MULE-4:UNAUTHORIZED
+
+
+[[ListDocuments]]
+== List Documents
+`<azure-cosmos-db-connector-mule-4:list-documents>`
+
+
+Returns a list of documents under the collection. This operation makes an HTTP GET request to the /dbs/{dbId}/colls/{collId}/docs endpoint.
+
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Db Id a| String |  User-generated name or ID of the database in which the stored procedure is created. |  | x
+| Coll Id a| String |  Name of the collection in which the stored procedure is created. |  | x
+| X ms consistency level a| Enumeration, one of:
+
+** STRONG
+** BOUNDED
+** SESSION
+** EVENTUAL |  Consistency level override. The valid values are listed in order of strongest to weakest. The override must be the same or weaker than the account's configured consistency level. |  |
+| X ms session token a| String |  String token used with session level consistency. Clients must echo the latest read value of this header during read requests for session consistency. |  |
+| A im a| String |  Must be set to *Incremental feed*, or omitted otherwise. Available for REST API version 2016-07-11 and later. |  |
```

Comment:
```suggestion
| A im a| String |  Must be set to `Incremental feed`, or omitted otherwise. Available for REST API version 2016-07-11 and later. |  |
```